SU-SWS/saa_victoria_subtheme
# ADAPT-000: Removed underline on cta and other icon links.

# READY FOR REVIEW

# Summary
- Remove underline from all types of icon links on regular state ( still underline on hover/focus)

# Review By (Date)
- 8/10

# Urgency
- low, but was requested

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. Test on a page with all kinds of links. This pr should: Remove underline from all types of icon links on regular state ( still underline on hover/focus)

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
